ingest-app/src/cmr/ingest/api/core.clj

```diff
@@ -233,6 +231,16 @@
   [concept]
   (pr-str (dissoc concept :metadata)))
 
+(defn log-concept-with-metadata-size
+  "Log the concept with metadata-size-bytes added, and metadata removed."
+  [concept request-context]
+  (let [metadata-size-bytes (-> (:metadata concept) (.getBytes "UTF-8") alength)
+        concept (assoc concept :metadata-size-bytes metadata-size-bytes)]
+    (info (format "Ingesting %s %s from client %s"
+                  (name (:concept-type concept)) 
+                  (concept->loggable-string concept)
```

Comment:
Yes. I originally just did (format "Ingest concept %s from client %s"...   because the concept->Loggable string already contains concept type. But then I'm not sure if anything is depending on the "Ingest granule", "Ingest collection" etc, so I just made it more specific.

Reply:
Okay. Catalino prefers to change it to (format "Successfully ingested concept %s from client %s"...  Done.